### PR TITLE
Release version v0.41.0

### PR DESCRIPTION
**Pull request by khanhtc1202**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/en/blog/releases/v0.41.0.md [hunk 1]
```diff
@@ -6,4 +6,68 @@ description: >
...
+
+### Internal Changes
+
+* Improve the health description for Cloud Run live state ([#4085](https://github.com/pipe-cd/pipecd/pull/4085))
+* Refactor the loadCloudRunManifest function ([#4089](https://github.com/pipe-cd/pipecd/pull/4089))
+* Enable to run lint/go on M1 ([#4088](https://github.com/pipe-cd/pipecd/pull/4088))
+* Implement lint/go command for local ([#4086](https://github.com/pipe-cd/pipecd/pull/4086))
+* Fix the wrong link for tools ([#4083](https://github.com/pipe-cd/pipecd/pull/4083))
+* Remove unused docs ([#4082](https://github.com/pipe-cd/pipecd/pull/4082))
+* Support Cloud Run application for planpreview ([#4079](https://github.com/pipe-cd/pipecd/pull/4079))
```
Comment (knanao) [inline docs/content/en/blog/releases/v0.41.0.md:42]: I suppose we should add this to the new features category.

Comment (khanhtc1202) [reply docs/content/en/blog/releases/v0.41.0.md:42]: Ooops, nice catch 🙏

File: docs/content/en/blog/releases/v0.41.0.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+### New features
+
+From now, you can manage your project's deployment via PipeCD deployment metrics. Please refer to the [docs](/docs/user-guide/insights/).
+
+![image](/images/insights.png)
+
+* Introduce Insights feature ([#4071](https://github.com/pipe-cd/pipecd/pull/4071))
+* Support setting kubectlVersion via piped config ([#4081](https://github.com/pipe-cd/pipecd/pull/4081))
+* Add `spec.trigger.onCommit.ignores` to application config ([#4059](https://github.com/pipe-cd/pipecd/pull/4059))
```
Comment (nghialv) [inline docs/content/en/blog/releases/v0.41.0.md:25]: @knanao PlanPreview and Health Status for Cloud Run application should be added here too, right?

Comment (knanao) [reply docs/content/en/blog/releases/v0.41.0.md:25]: Right, especially Planpreview is, but regarding health status whichever it's fine for me!

File: docs/content/en/blog/releases/v0.41.0.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+### Breaking Changes
+
+All __applications configuration__ file should NOT use the `spec.triggerPaths` (marked as deprecated previously), but use `spec.trigger.onCommit.paths` instead.
+
+* Remove trigger paths ([#4037](https://github.com/pipe-cd/pipecd/pull/4037))
+
+### New features
+
+From now, you can manage your project's deployment via PipeCD deployment metrics. Please refer to the [docs](/docs/user-guide/insights/).
```
Comment (nghialv) [inline docs/content/en/blog/releases/v0.41.0.md:37]: @khanhtc1202 I'm not sure about this `manage project's deployment`. 🤔

Comment (nghialv) [reply docs/content/en/blog/releases/v0.41.0.md:37]: Got it. How about `From now, you can view the deployment insights on your application and project. Please refer to ..`

Comment (khanhtc1202) [reply docs/content/en/blog/releases/v0.41.0.md:37]:
```
From now, you can have a better insight into your application/project's deployment via deployment data visualized by the PipeCD insight feature. Please refer to the [docs](/docs/user-guide/insights/).
```
thanks for comment, I think I will go with this 👀

**Review by nghialv — APPROVED** [review]

**Pull request merged**